Packs/SymantecDLP/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "Symantec Data Loss Prevention",
     "description": "Symantec Data Loss Prevention enables you to discover, monitor and protect your sensitive corporate information.",
     "support": "xsoar",
-    "currentVersion": "2.0.9",
+    "currentVersion": "3.0.0",
```

```suggestion
    "currentVersion": "2.1.0",
```

please raise by minor and not by major.

---

Change made in resubmitted content pack